.golangci.yml

```diff
@@ -85,8 +85,6 @@ linters-settings:
         files:
           - $all
         deny:
-          - pkg: "sync/atomic"
-            desc: "Use go.uber.org/atomic instead; see docs/dev/atomics.md"
```

Comment:
What do you think about removing `go.uber.org/atomic` in order to have a single way to handle atomic values?
Note: go.uber.org/atomic could be even marked as deprecated.

Reply:
I have no strong opinion. I think it generally makes sense to allow whatever libraries people want unless there's a specific safety or stability concern, which there's not for uber/atomic. I can also see the argument for having everyone use the same atomics package.